index.html

```diff
@@ -4303,7 +4303,7 @@ <h2>
         <td>biāodiǎn fúhào</td>
         <td>punctuation marks</td>
         <td>
-         <p its-locale-filter-list="en" lang="en">The marks that aid texts to record language, they are organic part of wrtting language, expressing pause, mood and special nature and functions of some parts (mainly words) in a sentence.</p>
+         <p its-locale-filter-list="en" lang="en">The marks that aid texts to record language, and are organic part of wrtting language. They are used to indicate pause, mood and special nature and functions of some parts (mainly words) in a sentence. (GB/T 15834-2011)</p>
```

Comment:
typo wrtting

Reply:
Thanks, Richard. I just pushed a commit to fix it.